### PR TITLE
[10.0][MIG] project_task_materials

### PR DESCRIPTION
**Pull request by chienandalu**
# Project Task Materials

Project Tasks allow to record the time spent on them, but some activities
often require you to keep a record of the material used also.

This module offers the ability to keep track of that material.

Note that only a simple record is made and no accounting or stock moves are
actually performed.

cc @Tecnativa

### CODE REVIEW

**Comment by angelmoya** [comment]
I Tested on runbot and reviewed code :+1:

**Comment by angelmoya** [comment]
this work was done on #232

**Comment by pedrobaeza** [comment]
It was a try from one subcontracted guy, but this one is the valid one.

File: project_task_material/README.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Usage
+=====
+
+To use this module:
+
+Every task form has a tab where product and their quantities can be added.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/9.0
```
Comment (dreispt) [inline project_task_material/README.rst]: s/9.0/10.0

File: project_task_material/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "version": "10.0.1.0.0",
+    "category": "Project Management",
+    "author": "Daniel Reis,"
+              "Tecnativa,"
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "product",
```
Comment (dreispt) [inline project_task_material/__manifest__.py]: `project` is missing

**Review by dreispt — COMMENTED** [review]
Why I can't see the changes to the original module?
It seems that even unchanged lines are part of this diff.

**Comment by chienandalu** [comment]
@dreispt Seems like I messed it up somehow with a rebase. I will try to fix it :/

**Comment by chienandalu** [comment]
@dreispt The diff is already fixed

File: project_task_material/models/project.py [hunk 3]
```diff
@@ -4,21 +4,21 @@
...
-        string='Materials used')
+        comodel_name='project.task.material', inverse_name='task_id',
+        string='Material used')
 
 
-class ProjectTaskMaterials(models.Model):
-    _name = "project.task.materials"
-    _description = "Task Materials Used"
+class ProjectTaskMaterial(models.Model):
+    _name = "project.task.material"
```
Comment (dreispt) [inline project_task_material/models/project.py]: If data table names are changed, we need a migration script.

File: project_task_materials [hunk 4]
```diff
@@ -0,0 +1 @@
+project_task_material
```
Comment (dreispt) [inline project_task_materials]: ?

File: project_task_material/models/project.py (hunk 3 above)
Comment (pedrobaeza) [reply project_task_material/models/project.py]: Migration script added.

**Review by pedrobaeza — APPROVED** [review]
@dreispt anything else?

**Review by dreispt — APPROVED** [review]

**Pull request merged**